### PR TITLE
Omniwallet mobile app second commit

### PR DESCRIPTION
**Pull request by mobilecheng**
Added main, market, my profile pages etc.

### CODE REVIEW

File: lib/view/main_view/home/omni_website.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class WebViewPage extends StatefulWidget {
+  @override
+  _WebViewPageState createState() => _WebViewPageState();
+}
+class _WebViewPageState extends State<WebViewPage> {
+  @override
+  Widget build(BuildContext context) {
+    return WebviewScaffold(
+      url: "https://www.omnigroup.com/",
```
Comment (achamely) [inline lib/view/main_view/home/omni_website.dart]: This website is not affiliate/related to omni. it is a completely different group. 
Omni is http://www.omnilayer.org/

File: lib/view/main_view/home/send_page.dart [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    return Scaffold(
+        key: this.key,
+        appBar: AppBar(title: Text(accountInfo.name+"转账"),),
+        body: this.body()
+    );
+  }
+
+  Widget makeRadioTiles() {
+    List<Widget> list = new List<Widget>();
+    list.add(RadioListTile<int>(value: 0,title: Text('慢速    0.00001 btc'), groupValue: _feeGroup, onChanged: _setvalue2));
```
Comment (achamely) [inline lib/view/main_view/home/send_page.dart]: This is ok to get started for the moment, but be advised the using a 'static' fee list is not advised. Fees should be calculated dynamically based on the number of utxo's the sending address is using as well as the current state of the network.   Using static fees can lead to potentially underfunded addresses in some cases and a transaction that either never confirms or takes too long to confirm

**Comment by achamely** [comment]
@mobilecheng looks good. only 1 change for the moment and 1 note to take into account for future development

File: lib/view/main_view/home/omni_website.dart (hunk 1 above)
Comment (mobilecheng) [reply lib/view/main_view/home/omni_website.dart]: Fixed

File: lib/view/main_view/home/send_page.dart (hunk 2 above)
Comment (mobilecheng) [reply lib/view/main_view/home/send_page.dart]: Added comment for the code.

**Pull request merged**